Sources/Core/API/TextureManager.swift

```diff
@@ -67,7 +76,22 @@ public final class TextureManager {
 
         guard let image = imageLoader.loadImageForTexture(named: name, scale: scale, format: format) else {
             guard scale > 1 else {
-                return nil
+              
+              #if DEBUG
+              let errorMessage = "Image with filename '\(name)' for a texture couldn't be found"
+              switch errorMode {
+              case .IgnoreError:
+                break
+              case .LogError:
+                print(errorMessage)
+              case .TriggerAssertOnError:
+                assertionFailure(errorMessage)
+              default:
```

Comment:
You don't need a default here, since you are already handing all cases. In fact, adding a default might be "dangerous", since if a new case is added we won't get a compiler warning that it's not handled 😱

Reply:
Oops 😓 Thanks for pointing that out. Never thought about it from a point of view of missing compiler warnings for new cases added. 💯 suggestion. Fixed.